src/utils/filter.py

```
@@ -0,0 +1,494 @@
+                                if validated_datetime is None:
+                                    logger.warning(
+                                        f"Skipping invalid datetime value in list: {val}"
+                                    )
+                                    continue
+                                casted_values.append(validated_datetime)
+                            else:
+                                casted_values.append(val)
+                        if casted_values:
+                            condition = column.in_(casted_values)
+                    else:
+                        condition = column.in_(op_value)
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "continue", remove the "else" and de-indent the code inside it

(R1724)